learning/tour-of-beam/terraform/cloud_functions/variables.tf

```diff
@@ -0,0 +1,46 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+variable "service_account_id" {
+  description = "The name of Service Account to run Cloud Function"
+}
+
+variable "project_id" {
```

Comment:
@olehborysevych this is the same approach as in Playground Terraform modular structure, we can drop usage of project_id variables sometimes, where possible (it will take project_id from provider), should we?